src/locales/en.json

```
@@ -344,6 +344,26 @@
         "Atomic": "Bug corrected in the periodic table",
         "Gutter": "Gutter corrected in the python application"
       }
+      "1-0-1": {
+        "XNT" : "XNT Cyclic (Multiple presses of XNT change the character inserted between X, N, T and θ)",
+        "NewPol" : "Changed font",
```

Can you rename to `NewFont` and edit the text to `New font` (I prefer this form, but it's just me)
```suggestion
        "NewPol" : "New font",
```